### PR TITLE
[FCP-569] As a partner i want the identity provider login screen to display a default eidas level corresponding to the eidas level set by the service provider

### PR DESCRIPTION
**Pull request by OlivierDelalande**
inside router.ejs, on route get /interaction/:grant, adding a constant acr coming from req.session.info.acr_values, and passing to the render of sign-in.
Inside sign-in setting the default value of the dropdown to the value acr coming from the controller if exist, or to eidas1.

### CODE REVIEW

File: src/router.js [hunk 1]
```diff
@@ -30,11 +30,13 @@ export const mountRoutes = (app, provider) => {
       const notifications = messages[req.query.notification]
         ? [messages[req.query.notification]]
         : [];
+      const acr = req.session.infos.acr_values;
```
Comment (rdubigny) [inline src/router.js]: what if infos is not defined ?

image

File: src/router.js [hunk 2]
```diff
@@ -30,11 +30,14 @@ export const mountRoutes = (app, provider) => {
       const notifications = messages[req.query.notification]
         ? [messages[req.query.notification]]
         : [];
-
+ 
+      const acr = req.session.infos.acr_values ? req.session.infos.acr_values : '';
```
Comment (rdubigny) [inline src/router.js]: This will still crash if infos is not defined...

File: src/router.js (hunk 1 above)
Comment (OlivierDelalande) [reply src/router.js]: yes handled in further commits

File: src/router.js (hunk 2 above)
Comment (OlivierDelalande) [reply src/router.js]: handled, but you commented before i pushed a new version

**Comment by rdubigny** [comment]
@OlivierDelalande can you rename the PR ? (with reference to the FC ticket for example) . "working feature" is too generic for a PR name or commit name. 

If you do not want to rename your commit (which can be painful, I agree). We should, at least, squash the commit when we merge this.

File: src/router.js [hunk 3]
```diff
@@ -31,10 +31,15 @@ export const mountRoutes = (app, provider) => {
         ? [messages[req.query.notification]]
         : [];
 
+      const {
+        session: { info: { acr_values: acr } = {} },
```
Comment (rdubigny) [inline src/router.js]: image

do you need help to test this ?

Comment (OlivierDelalande) [reply src/router.js]: As seen together i  have been able to test on my hand, feel free to tell me if you could on yours.

**Comment by rdubigny** [comment]
it breaks in production 
image

I revert it

**Pull request merged**